app/views/providers/change_of_names_interrupts/show.html.erb

```diff
@@ -1,14 +1,14 @@
-<div class="interruption-panel">
-  <%= page_template(page_title: t(".title_html"), column_width: "full", template: :basic) do %>
-    <%= page_heading(size: "l") %>
+<div class="moj-interruption-card">
+  <div class="moj-interruption-card__content">
+    <%= page_template(page_title: t(".title_html"), template: :basic) do %>
+      <%= page_heading(size: "l", classes: "moj-interruption-card__heading") %>
 
-    <div class="maximize-text-width">
-      <p class="govuk-body"><%= t(".paragraph") %></p>
-    </div>
+      <p class="govuk-body moj-interruption-card__body"><%= t(".paragraph") %></p>
 
-    <div class='govuk-button-group'>
-      <%= govuk_button_link_to t(".change"), providers_legal_aid_application_change_of_names_path, inverse: true, role: "button" %>
-      <%= govuk_button_link_to t(".applications"), in_progress_providers_legal_aid_applications_path, inverse: true, role: "button" %>
-    </div>
-  <% end %>
+      <div class='govuk-button-group'>
```

Comment:
Based on the component, this seems to be missing a class `moj-interruption-card__actions`